### DISCUSSION TITLE
Plotman Status doesnt show status for job where -n 2 is used

### DISCUSSION THREAD

**Discussion by nomadengineer** [ericaltendorf/plotman#396] [General]
Hi,

Plotman Status does not display proper status when I launch plotters with the `-n 2` option.

```
(venv) ubuntu@ip-172-31-49-241:~/chia$ plotman status
 plot id    k                                tmp                                   dst    wall   phase   tmp     pid   stat     mem    user    sys     io
c6f84896   32   /home/ubuntu/chia/plot-temp/tmp4   /home/ubuntu/chia/farm-vol2/farm-1/   11:00     4:0     0   91626    SLP   10.0G   10:14   0:38   1:52
181500cb   32   /home/ubuntu/chia/plot-temp/tmp3   /home/ubuntu/chia/farm-vol2/farm-1/   11:10     4:0     0   91191    SLP   10.0G   10:28   0:39   1:51
bfd17e93   32   /home/ubuntu/chia/plot-temp/tmp2   /home/ubuntu/chia/farm-vol2/farm-1/   11:20     4:0     0   90774    SLP    9.9G   10:39   0:40   1:47
```
Or is it fixed in a newer version? :p

Thanks!

**Comment by altendky** [comment]
What is it that you think is improper there?  Sorry if I am missing something obvious.

**Reply by nomadengineer** [reply]
My plotting jobs were started with the `-n 2` option, which means the same plotting process will generate 2 plots in sequence.
Now Plotman gives the correct phase status lile 1:7, 3:5 etc when the first job is going on. When the 2nd job starts it is added in the same log file and hence plotman only shows 4:0 of the first job.
I hope I was able to explain it properly!

**Reply by altendky** [reply]
Sorry, yes, I didn't have my head straight on which option `-n` was...  `:[`  Is this just a thing while your manually launched jobs finish out or is there some reason you want to continue running this way?  I think it is unlikely that this gets fixed in plotman soon just based on the workload with other tasks that seem likely to be a more common need.  Sorry.  `:[`  But, it would be good for me to at least understand the use case.
